src/test/run-make-fulldeps/coverage-reports/expected_show_coverage_counters.async.txt

```diff
@@ -28,11 +28,8 @@ Counter in file 0 79:14 -> 79:16, 0
 Counter in file 0 81:1 -> 81:2, 0
 Counter in file 0 91:25 -> 91:34, 0
 Counter in file 0 5:1 -> 5:25, #1
-Counter in file 0 5:25 -> 6:14, #1
```

Comment:
The changes in the counters file in async.txt are just another example of the non-deterministic ordering of output from `llvm-cov`.

These 4 lines (from 31-34) moved to line 50 when I added support for multiple `SwitchInt` targets to the same `BasicBlock`.

The other lines moved after I updated `mod.rs` to ignore MIR that starts with `Unreachable`.

I don't think these changes should have had any real effect on the existing test results, and they don't except in this one test sample, and only in the debug counters.

The `Makefile` based test does not fail based on differences in counter files. So it's not a problem, but it would be nice to make these more deterministic in the future, perhaps by sorting the entire file before comparing.